nautobot/core/api/metadata.py

```diff
@@ -65,10 +65,15 @@ def get_field_info(self, field):
         return field_info
 
 
-class StatusFieldMetadata(BulkOperationMetadata):
+class StatusFieldMetadata(ContentTypeMetadata):
     """Emit `Status` serializer fields as a choice enum."""
 
     def get_field_info(self, field):
+        # Keep backwards compatible behavior if serializer also has `ContentTypeField` fields specified
+        if isinstance(field, ContentTypeField):
+            return super().get_field_info(field)
```

Comment:
Is this addition actually necessary? We already call `super().get_field_info(field)` below and eventually return it.

Reply:
I was thinking of just returning right away rather than hitting the logic below, but it will be removed in an upcoming commit.